Common/include/geometry/CGeometry.hpp

```diff
@@ -910,6 +911,20 @@ class CGeometry {
    */
   inline virtual void SetRestricted_GridVelocity(const CGeometry *fine_grid) {}
 
+  /*!
+   * \brief Compute the surface area of all global markers.
+   * \param[in] config - Definition of the particular problem.
+   */
+  void ComputeSurfaceAreaCfgFile(const CConfig *config);
+
+  /*!
+  * \brief Adapt Heatflux value for integrated heatflux.
+  * \param[in,out] Wall_HeatFlux - Heatflux in [W] which is to be adapted to [W/m^2].
+  * \param[in] config - Definition of the particular problem.
+  * \param[in] val_marker - Surface marker where the boundary condition is applied.
+  */
+  void UpdateIntegrated_Heatflux(su2double* Wall_HeatFlux, unsigned short val_marker, const CConfig *config) const;
```

Comment:
This would have made sense in the solver, maybe... but in geometry it is better to deal in geometry only.
```suggestion
  su2double GetSurfaceArea(const CConfig *config, unsigned short val_marker) const;
```
Also don't start passing scalars by pointer for the love of god :smile:

Reply:
... gotta save those 8 bytes :floppy_disk:

Reply:
8794f14  here you go